src/base/http/types.h

```diff
@@ -108,7 +108,11 @@ namespace Http
         uint code;
         QString text;
 
-        ResponseStatus(uint code = 200, const QString &text = "OK"): code(code), text(text) {}
+        ResponseStatus(uint code = 200, const QString &text = "OK")
```

Comment:
Now you can drop this constructor at all and create it as aggregate `{200, "Ok"}`.

Reply:
done.